### ISSUE TITLE
Specs satisfying the `STULabel` dependency were found, but they required a higher minimum deployment target.

### ISSUE THREAD

**Issue by danielgek** [Akylas/nativescript-label#2]
After the 1.0.5 update, i'm getting this error, but as far as i can see you are not using it so i guess it can be removed from the podfile?

The Error: 
```
[!] CocoaPods could not find compatible versions for pod "STULabel":
  In Podfile:
    STULabel

Specs satisfying the `STULabel` dependency were found, but they required a higher minimum deployment target.
```

**Comment by farfromrefug** [comment]
fixed in 1.0.6. That dep was actually not used :s

**Comment by danielgek** [comment]
Awesome, Thanks man ;)
